sunxspex/photon_power_law.py

```diff
@@ -0,0 +1,100 @@
+import functools
+
+import numpy as np
+
+import astropy.units as u
+
+PHOTON_RATE_UNIT = u.ph / u.keV / u.cm**2 / u.s
+
+
+@u.quantity_input
+def power_law_integral(
+    energy: u.keV,
+    norm_energy: u.keV,
+    norm: PHOTON_RATE_UNIT,
+    index: u.one
+) -> (PHOTON_RATE_UNIT * u.keV):
+    """Evaluate the antiderivative of a power law at a given energy.
```

Comment:
This looks great @settwi ! Can the docstrings be changed to be more similar to the other modules that roughly follow the Numpy style guide? E.g., 
```
"""
Description

Parameters
-----------
...
Returns
--------
...
"""```